src/stream/src/executor_v2/hash_agg.rs

```diff
@@ -292,93 +293,6 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
         Ok(())
     }
 
-    async fn flush_data(
```

Comment:
In fact, we can make flush_data returns an impl Stream by add try_stream macro on it.